elastic/integrations
# [Azure OpenAI] Fix documentation navigations & enable log datastream

- Bug

- This PR fixes the documentation tabs for logs and metrics.
- Enables the logs datastream selection by default.


## Checklist



## How to test this PR locally

Update to the latest version and verify the left navigation has appropriate information for logs and metrics.


## Screenshots

### Before
<img width="1728" alt="before-docs" src="https://github.com/elastic/integrations/assets/101238137/4d336f11-4ecf-46a1-be16-29d0a97f66a7">

### After
<img width="1728" alt="after-docs" src="https://github.com/elastic/integrations/assets/101238137/eb9d612f-5604-4423-98f8-54dda80ac9da">
